### PR TITLE
update version and changelog for pypi

### PR DESCRIPTION
**Pull request by zhangqiaorjc**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,9 +8,13 @@ Remember to align the itemized text with the first line of an item within a list
...
+## jax 0.2.25 (Unreleased)
+* [GitHub
+  commits](https://github.com/google/jax/compare/jax-v0.2.25...main).
+
 ## jaxlib 0.1.74 (Unreleased)
 
-## jax 0.2.25 (Unreleased)
+## jax 0.2.25 (Nov 10, 2021)
 * [GitHub
   commits](https://github.com/google/jax/compare/jax-v0.2.24...main).
```
Comment (yashk2810) [inline CHANGELOG.md:19]: update the main to jax-v0.2.25
```suggestion
  commits](https://github.com/google/jax/compare/jax-v0.2.24...jax-v0.2.25).
```

Comment (zhangqiaorjc) [reply CHANGELOG.md]: amended

**Review by yashk2810 — APPROVED** [review]

File: jax/version.py [hunk 2]
```diff
@@ -12,5 +12,5 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-__version__ = "0.2.25"
+__version__ = "0.2.26"
```
Comment (jakevdp) [inline jax/version.py:15]: We want this updated after the release commit, yes?

Comment (jakevdp) [reply jax/version.py:15]: Oh, my mistake - this is correct, but the `0.2.25 (Unreleased)` above should be `0.2.26 (Unreleased)` – that threw me off.

Comment (zhangqiaorjc) [reply jax/version.py:15]: yes, updated, thanks for the catch

**Pull request merged**